newsletter_automation/newsletter/routes.py

```diff
@@ -126,6 +124,7 @@ def add_articles():
 @Authentication_Required.requires_auth
 def articles():
     """To add articles through pages"""
+    #app.logger.info(request.path)
```

Comment:
Please delete this commented out line of code. In general, there should be no commented out lines of code.

Reply:
I have removed all the commented out codes from routes file